cis/system/hsp-cis-1-1-3-controller-manager-pod-block.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-cis-1-1-3-controller-manager-pod-block
+spec:
+  tags: ["CIS", "1.1.3","K8"]
```

Comment:
Keep the same tag throughout the repo, kindly use `k8s` instead of `K8`

Reply:
updated.